lib/invoice.rb

```diff
@@ -2,15 +2,13 @@ class Invoice
   attr_reader :id,
               :customer_id,
               :merchant_id,
-              :status,
-              :created_at,
-              :updated_at
```

Comment:
Should we be dropping time attr_readers for the other classes too, for spec harness?

Reply:
Yes! We will need to do that. Good catch! Will update.

Reply:
@NoahZinter yea, check out my Wiki on the spec harness. Need to allow for different inputs, so the attr_readers are duplicative